src/loader.c

```diff
@@ -1954,8 +2059,7 @@ static void ldr_show_pw_line(struct db_main *db, char *line)
 	if (found && show) {
 		if (source[0])
 			printf("%c%s", db->options->field_sep_char, source);
-		else
-			putchar('\n');
+		putchar('\n');
 	}
```

Comment:
Oh right, I thought it changed from fgetl to fgetll. But it actually used fgets before. That explains it! Hard to see with naked eye when mind was decided.